mobile/src/lib/network.js

```diff
@@ -0,0 +1,12 @@
+export const watchNetworkState = (onConnectionChange) => {
+  document.addEventListener('offline', onConnectionChange, false)
+  document.addEventListener('online', onConnectionChange, false)
+}
+
+export const hasCordovaPlugin = window.navigator.connection !== undefined && window.Connection !== undefined
+
+export const getConnectionType = () => hasCordovaPlugin ? navigator.connection.type : 'wifi'
```

Comment:
If I check `hasCordovaPlugin` to get either the `navigator.connection.type` value or a constant value, it is because we don't have `Connection.WIFI` outside of cordova. And I want to build a lib that can work outside cordova environment (first for development time, but also for tomorrow when mobile navigator will have support for that kind of things... Firefox OS we miss you).